apps/src/constants.js

```diff
@@ -44,6 +44,7 @@ exports.TestResults = {
 
   // start using negative values, since we consider >= 20 to be "solved"
   EMPTY_FUNCTION_NAME: -1,       // We have a variable or function with the name ""
+  NESTED_FOR_SAME_VARIABLE: -2,
```

Comment:
I realized while doing this that -1 means both NO_TESTS_RUN and EMPTY_FUNCTION_NAME. We could change EMPTY_FUNCTION_NAME, but that would then invalidate any current instances we have stored in the db.